lib/srv/ctx.go

```diff
@@ -158,6 +158,9 @@ type IdentityContext struct {
 
 	// RouteToCluster is derived from the certificate
 	RouteToCluster string
+
+	// ActiveRequests is a string of active access request ID's
+	ActiveRequests string
```

Comment:
May multiple access request IDs be stored in this string? The wording of the comment indicates it does but it is neither an array nor specify the layout of the string.

Reply:
Yes, multiple access requests can be stored in this field. I will update the comment to specify what the layout of this string will look like.